src/schema/me/followed_shows.ts

```diff
@@ -0,0 +1,44 @@
+import { ShowType } from "schema/show"
+import { IDFields } from "schema/object_identification"
+
+import { pageable, getPagingParameters } from "relay-cursor-paging"
+import { connectionDefinitions, connectionFromArraySlice } from "graphql-relay"
+import { GraphQLObjectType } from "graphql"
+
+export const FollowedShowType = new GraphQLObjectType({
+  name: "FollowedShow",
+  fields: {
+    partner_show: {
+      type: ShowType,
+    },
+    ...IDFields,
+  },
+})
+
+export default {
+  type: connectionDefinitions({ nodeType: FollowedShowType }).connectionType,
```

Comment:
This is ok, but an alternative is to model this similar to 'partner artists'. That is, a partner can have an artist, where the connection should return a node of an `Artist` type, and not a `PartnerArtist` type, however the _edges_ connecting a partner to an artist node, is made up of the `PartnerArtist` data (such as representation status, etc.)

Similarly here, a user/me should be connected to `Show`, since _that's_ the actual interesting object. A `FollowShow` is a relatively uninteresting/meta type, that clients shouldn't have to 'know' about. (What if we change following to some other behaviors, we could still preserve a user -> show graph, but not user -> followshow )

An example definition is https://github.com/artsy/metaphysics/blob/20b27b343ca809a9c2f88b329423f9a86e1d27b0/src/schema/partner_artist.ts#L88-L97 and used https://github.com/artsy/metaphysics/blob/master/src/schema/artist/highlights.ts#L13

This lets you have a connection of partners on an artist (artist -> partners), and the edge between each node contains metadata about that relationship. I would think that for follow shows, a similar setup could work (connection is me -> shows, and the edge contains metadata about that, was it a follow? an auto-follow? some other reason, etc.)

This is totally not a blocker to your setup, feel free to carry on with that! Just wanted to point out an alternative.

Reply:
I think that's what @alloy was alluding to https://github.com/artsy/gravity/pull/12060#issuecomment-448696507 , where the follows are the edges and not the nodes.

Reply:
Yup, that's what I meant! Thanks for the longer explanation @mzikherman 👍

Reply:
I believe that the current configuration satisfies everything outlined in this conversation. I'm opening this up for review now and will tag everyone who commented here, to sanity check me. I'm going to include screenshots of GraphiQL input and output just as the demonstration of functionality, so I'll update this thread with those and then I'll assign.